src/App.jsx

```diff
@@ -1,107 +1,14 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
+import postsFromServer from './api/posts.json';
+import { PostList } from './components/PostList';
 // import commentsFromServer from './api/comments.json';
 // import usersFromServer from './api/users.json';
 
```

Comment:
Consider making a normalized array of objects by mapping postsFromServer and adding a user and comments to each post